### PR TITLE
Fixed deprecation issue and Bug fix for signal to inject multiple target pods together and fix for Target Pods ENV not Set

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

 - Fixing depreciation issue
 - Updated with litmus-go master 
 - Updating minor code base
 - Adding support for multiple target pods to inject chaos
 - Fix for Target Pods ENV not Set

### CODE REVIEW

File: contribute/developer-guide/README.md [hunk 1]
```diff
@@ -238,7 +238,8 @@ Follow the steps provided below to setup okteto & test the experiment execution.
       'experiments/sample_category/sample_exec_chaos',
       'experiments/sample_category/sample_exec_chaos/experiment',
       ```
-    - Add `&` operator in the end of chaos-inject command `CHAOS_INJECT_COMMAND` example: `md5sum /dev/zero &`. As we are running chaos command as a background process in a separate thread.  
+    - Add `&` operator at the end of chaos commands `CHAOS_INJECT_COMMAND` and `CHAOS_KILL_COMMAND` example: `md5sum /dev/zero &`.
```
Comment (ispeakc0de) [inline contribute/developer-guide/README.md:241]: Do we need `&` for CHAOS_KILL_COMMAND?

File: pkg/utils/common/pods.py [hunk 2]
```diff
@@ -128,7 +128,6 @@ def GetTargetPodsWhenTargetPodsENVNotSet(self, podAffPerc , nonChaosPods, chaosD
 		filteredPods = []
 		realPods = []
 		for pod in nonChaosPods.items:
-		
 			parentName, err = annotation.GetParentName(clients, pod, chaosDetails)
```
Comment (ispeakc0de) [inline pkg/utils/common/pods.py:131]: Can we derive parent name only if annoationCheck is true?

File: pkg/utils/common/pods.py [hunk 3]
```diff
@@ -144,16 +143,15 @@ def GetTargetPodsWhenTargetPodsENVNotSet(self, podAffPerc , nonChaosPods, chaosD
 					filteredPods.append(pod)
 					logging.info("[Info]: chaos candidate of kind: %s, name: %s, namespace: %s", chaosDetails.AppDetail.Kind, parentName, chaosDetails.AppDetail.Namespace)
 			else:
-				for pod in nonChaosPods.items:
-					filteredPods.append(pod)
+				filteredPods.append(pod)
 				logging.info("[Info]: chaos candidate of kind: %s, name: %s, namespace: %s", chaosDetails.AppDetail.Kind, parentName, chaosDetails.AppDetail.Namespace)
```
Comment (ispeakc0de) [inline pkg/utils/common/pods.py:147]: we can remove this log

File: pkg/utils/common/pods.py [hunk 4]
```diff
@@ -144,16 +143,15 @@ def GetTargetPodsWhenTargetPodsENVNotSet(self, podAffPerc , nonChaosPods, chaosD
...
 		
 		if len(filteredPods) == 0:
 			return client.V1PodList(items=filteredPods), ValueError("No target pod found")
 			
 		newPodListLength = max(1, maths.Adjustment(podAffPerc, len(filteredPods)))
```
Comment (ispeakc0de) [inline pkg/utils/common/pods.py:152]:
```suggestion
		newPodListLength = max(1, maths.Adjustment(min(podAffPerc,100), len(filteredPods)))
```

File: pkg/utils/common/pods.py (hunk 2 above)
Comment (ispeakc0de) [reply pkg/utils/common/pods.py:131]: change
return isPodsAvailable, None --> return isPodsAvailable, ValueError("{} pod is not available in {} namespace".format(podList[index], namespace)
 https://github.com/litmuschaos/litmus-python/pull/23/files#diff-d887f697d1fd33d3db83a05b0285cd627948aa70f5789b6bafee2084d08ab6f1R31

File: contribute/developer-guide/README.md (hunk 1 above)
Comment (oumkale) [reply contribute/developer-guide/README.md:241]: Yes, we need it for multiple target pods, Like to get execute and to get back to another target thread.

File: pkg/utils/common/pods.py (hunk 3 above)
Comment (oumkale) [reply pkg/utils/common/pods.py:147]: Sure,

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**